src/screens/Main.tsx

```diff
@@ -34,14 +34,67 @@ interface Props {
   className?: string;
 }
 
+interface MenuActionItemsProps {
+  idx: number;
+  title: string;
+  isEnabled: boolean;
+  component: React.ReactElement;
+}
+
+const MenuContents = [
+  {
+    idx: 0,
+    title: 'Transfer',
+    isEnabled: true,
+    component: (
+      <>
+        <Transfer />
+        <ExtensionAccountCheck component={<Accounts />} />
+      </>
+    )
+  },
+  {
+    idx: 1,
+    title: 'Remark',
+    isEnabled: true,
+    component: <Remark />
+  },
+  {
+    idx: 2,
+    title: 'Custom Call',
+    isEnabled: true,
```

Comment:
Components might  be enabled/disabled depending on runtime availability (but it's more a Milestone 3 kind of work).

Reply:
In the case of "Connect to a wallet" Im not sure if we should "disable" the component when extension exists.